README.md

```diff
@@ -74,6 +74,16 @@ To upgrade a plugin (for example, to address a security vulnerability), tick the
 Jenkins should restart during a quiet period when no jobs are running (the restart will take a few seconds).
 
 
+## Jobs
+
+Our Jenkins jobs are generated on the Jenkins box using the `jenkins-job-builder` tool via Ansible. If you want to test your job file locally before uploading, you can use
+
+```bash
+$ make test [JOBS=<job_name>]
+```
+
+If you want to test all of the job files it can be a bit slow, which is why `make test` will try and pre-cache the output from Ansible the first time you run it.
```

Comment:
This is slightly unclear - will a regular `make test` test all jobs then?

Reply:
Yes, it will.

What I was trying to document is that if you run `make test JOBS=<job_name>` and you haven't run it before it will copy all of the Ansible files for you, which might be slightly surprising behaviour.

It does this because if you run `make -C test` without a cache, ansible will run once for each file, which takes a long time.

Reply:
It's not great behaviour, but again it can be fixed later.

Reply:
Ohhhhhkay.

Reply:
The Makefile is kind of a mess tbh, if you want we can pair on it.